src/display/api.js

```diff
@@ -953,8 +959,8 @@ class PDFDocumentProxy {
   }
 
   /**
-   * @returns {Promise<TypedArray>} A promise that is resolved with a
-   *   {TypedArray} that has the raw data from the PDF.
+   * @returns {Promise<Uint8Array>} A promise that is resolved with a
+   *   {Uint8Array} that has the raw data from the PDF.
```

Comment:
While unrelated to the rest of the patch, this method should always be returning a `Uint8Array` hence it cannot hurt to be more specific in the JSDocs.